apps/portal/extensions/providers/rdbms/api.js

```diff
@@ -89,23 +99,28 @@ var getConfig, validate, getMode, getSchema, getData, registerCallBackforPush;
     getSchema = function (providerConfig) {
         var db = null;
         try {
-            var databaseUrl = providerConfig['db_url'];
+            var databaseUrl = providerConfig['dbUrl'];
             var databaseName = databaseUrl.substring(databaseUrl.lastIndexOf("/") + 1);
-            var tableName = providerConfig['table_name'];
+            var tableName = providerConfig['tableName'];
             var db_query = '';
```

Comment:
yes. We use google javascript style guide, I have changed all the names to camel case according to the guide.